Microsoft.Maui-mac.slnf

```diff
@@ -42,6 +42,7 @@
       "src\\Graphics\\src\\Graphics\\Graphics.csproj",
       "src\\Graphics\\src\\Text.Markdig\\Graphics.Text.Markdig.csproj",
       "src\\Graphics\\tests\\DeviceTests\\Graphics.DeviceTests.csproj",
+      "src\\Graphics\\tests\\Graphics.Tests\\Graphics.Tests.csproj",
```

Comment:
this is needed because build needs this to be built to run tests against the dll after